scala/README.md

```diff
@@ -868,7 +880,9 @@ Additional Remarks
 
     [Martin Fowler](http://www.martinfowler.com/bliki/AnemicDomainModel.html)
 
-0. Always remember Tip 4: "Do not Live with Broken Windows: Fix each as soon as it is discovered." - The Pragmatic Programmer, Andrew Hunt and David Thomas.
+0. Always remember Tip 4: "[Do not Live with Broken Windows](http://www.artima.com/intv/fixit.html): Fix each as soon as it is discovered." - The Pragmatic Programmer, Andrew Hunt and David Thomas.
+
+Part II, [Orthogonality and the DRY Principle](http://www.artima.com/intv/dry.html), is also an interesting reading. [Full interview](http://www.artima.com/intv/plain.html).
```

Comment:
"is also an interesting read*" or "is also interesting reading" (the former is more common)